lib/Runtime/ByteCode/ByteCodeReader.cpp

```diff
@@ -157,7 +154,18 @@ namespace Js
 
     OpCode ByteCodeReader::PeekByteOp(const byte * ip)
     {
-        return (OpCode)*ip;
+        return ReadByteOp(ip);
+    }
+
+    OpCode ByteCodeReader::ReadExtOp(const byte*& ip)
+    {
+        uint16*& extIp = (uint16*&)ip;
```

Comment:
>(uint16*&)ip [](start = 25, length = 12)

The & on the RHS should be unnecessary since ip is already an alias and you're assigning to an alias.